pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/invertedindex/MultiColumnRealtimeLuceneTextIndex.java

```diff
@@ -149,8 +150,65 @@ public MutableRoaringBitmap getDocIds(String searchQuery) {
     throw new UnsupportedOperationException("Multi-column text index requires column name to query!");
   }
 
+  @Override
+  public MutableRoaringBitmap getDocIds(String column, String searchQuery, @Nullable String optionsString) {
+    if (optionsString != null && !optionsString.trim().isEmpty()) {
+      LuceneTextIndexUtils.LuceneTextIndexOptions options = LuceneTextIndexUtils.createOptions(optionsString);
+      Map<String, String> optionsMap = options.getOptions();
+      if (!optionsMap.isEmpty()) {
+        return getDocIdsWithOptions(column, searchQuery, optionsMap);
```

Comment:
Let's pass `LuceneTextIndexUtils.LuceneTextIndexOptions` all the way down to `LuceneTextIndexUtils.createQueryParserWithOptions()`